doc/lua_api.txt

```diff
@@ -3295,7 +3295,11 @@ This is basically a reference to a C++ `ServerActiveObject`
 * `set_wielded_item(item)`: replaces the wielded item, returns `true` if successful
 * `set_armor_groups({group1=rating, group2=rating, ...})`
 * `get_armor_groups()`: returns a table with the armor group ratings
-* `set_animation({x=1,y=1}, frame_speed=15, frame_blend=0, frame_loop=true)`
+* `set_animation(frame_range, frame_speed, frame_blend, frame_loop)`
+  * `frame_range`: table {x=num, y=num}, default: `{x=1.0, y=1.0}`
+  * `frame_speed`: float, default: `15.0`
+  * `frame_blend`: float, default: `0.0`
+  * `frame_loop`: boolean, default: `true`
```

Comment:
These values need to be checked for if they are actually defaults

Reply:
l've looked at script/ to find the defaults. It looks like every parameter is optional.

Reply:
Ok.

Reply:
`float` -> `num` or `number`, as used above.

Reply:
l wrote float because the double is converted to a 32 bit float point number.